### ISSUE TITLE
Handle files with Windows line endings

### ISSUE THREAD

**Issue by chrisoldwood** [pbthorste/avtool#2]
One of the problems with the real `ansible-vault` tool is that it doesn't like Windows line endings, i.e. a `CR`+`LF` at the end of every line. Hence you have to use _another_ tool like `dos2unix` on the encrypted file first or you'll get an obscure error. The same occurs with `avtool`, you get:
```
> avtool view secrets-file.yml
Enter password: ERROR: runtime error: index out of range 
```

It would be really nice if this extra step could be avoided and `avtool` be agnostic on line endings.

**Comment by pbthorste** [comment]
Shouldn't be a problem, I'll look into it :)

**Comment by pbthorste** [comment]
fixed in version 1.1.1
